app/views/assessments/bulkGrade.html.erb

```diff
@@ -13,16 +13,16 @@
   <% end %>
 <% end %>
 
-  <%= render "bulkGrade_entries", :entries => @entries, :problems => @assessment.problems if @entries %>
-
   <% if @valid_entries %>
+    <%= render "bulkGrade_entries", :entries => @entries, :problems => @assessment.problems if @entries %>
     <%= form_for :confirm, :url => "bulkGrade_complete", :html => { :class => "confirm" } do |f| %>
       <%= f.hidden_field :bulkGrade_csv, :value => @csv %>
       <%= f.hidden_field :bulkGrade_data_type, :value => @data_type %>
 
-      <%= f.submit 'Yes' , {:class=>"btn submit"} %><%= link_to "No", { :action => :bulkGrade }, :class => "no" %>
+      <%= f.submit 'Yes' , {:class=>"btn submit"} %><%= link_to "No", { :action => :bulkGrade }, :class => "btn submit" %>
```

Comment:
Before with class => "no", the buttons looked like: 
![image](https://user-images.githubusercontent.com/31053044/77702592-e36b9200-6f86-11ea-9fb1-5daaecf84db3.png)

Now they look like: 

![image](https://user-images.githubusercontent.com/31053044/77702646-0007ca00-6f87-11ea-8b24-9c1fc523c94f.png)